src/scripts_manager/scripts_manager_api.py

```
@@ -0,0 +1,33 @@
+                logging.error(f"Script not found: {script_name}")
+                return ""
+            script_file = name_to_scripts[script_name]
+            return self.scripts_manager_service.get_script_description(script_file)
+        except Exception as ex:
+            logging.exception(f"Error getting script description for {script_name}: {ex}")
+            return ""
+                
+    def execute_script(self, script_name, additional_text, flags):
+        try:
+            return self.scripts_manager_service.execute_script(script_name, additional_text, flags)
+        except Exception as ex:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)